src/lib/priceEncoder.ts

```diff
@@ -0,0 +1,28 @@
+import { BigNumber } from "ethers";
+import bn from "bignumber.js";
+
+bn.config({ EXPONENTIAL_AT: 999999, DECIMAL_PLACES: 18 });
+
+function bigNumberToBig(val: BigNumber, decimals: number): bn {
```

Comment:
@WarTech9 I changed this too, `decimals` no longer defaults to `18`, is always mandatory to prevent mistakes.

Reply:
```suggestion
function bigNumberToBnNative(val: BigNumber, decimals: number): bn {
```